.pre-commit-config.yaml

```diff
@@ -91,3 +91,10 @@ repos:
     - requests_unixsocket
     - six
     - Sphinx
+
+
+-   repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Could you move this before `pylint`? It's faster and pylint is quite slow, this is why it needs to be the last one for best linting responsiveness.
Also, use the same indentation style as the entries above — with just one space after `-`, not 3.